src/mpc-hc/MainFrm.cpp

```diff
@@ -1219,6 +1244,26 @@ void CMainFrame::RecalcLayout(BOOL bNotify)
     }
 }
 
+void CMainFrame::EnableDocking(DWORD dwDockStyle) {
+    ASSERT((dwDockStyle & ~(CBRS_ALIGN_ANY | CBRS_FLOAT_MULTI)) == 0);
+
+    m_pFloatingFrameClass = RUNTIME_CLASS(CMiniDockFrameWnd);
+    for (int i = 0; i < 4; i++) {
+        if (dwDockBarMap[i][1] & dwDockStyle & CBRS_ALIGN_ANY) {
+            CMPCThemeDockBar* pDock = (CMPCThemeDockBar*)GetControlBar(dwDockBarMap[i][0]);
+            if (pDock == NULL) {
+                pDock = new CMPCThemeDockBar;
+                if (!pDock->Create(this,
+                    WS_CLIPSIBLINGS | WS_CLIPCHILDREN | WS_CHILD | WS_VISIBLE |
+                    dwDockBarMap[i][1], dwDockBarMap[i][0])) {
+                    AfxThrowResourceException();
+                }
+            }
+        }
+    }
+    //CFrameWnd::EnableDocking( dwDockStyle);

```

Comment:
Remove commented out code, unless good reason to keep it

Reply:
done.